src/test/compile-fail/feature-gate-negate-unsigned.rs

```diff
@@ -16,7 +16,7 @@ impl std::ops::Neg for S {
     fn neg(self) -> u32 { 0 }
 }
 
-const _MAX: usize = -1;
+const _MAX: usize = -(2 - 1);
```

Comment:
MIR errors while building while the test expects an error emitted during lint-checking (but not suppressible AFAICT), and the driver stops on errors somewhere in between.